<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1051</width>
    <height>672</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Detector de Vehiculos Aerio</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(231, 241, 255);</string>
  </property>
  <widget class="QWidget" name="mainWindow">
   <widget class="QGroupBox" name="addline">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>0</y>
      <width>181</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true"/>
    </property>
    <property name="title">
     <string>Añadir lineas de detección</string>
    </property>
    <widget class="QToolButton" name="addVLine">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>26</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
    <widget class="QToolButton" name="addHLine">
     <property name="geometry">
      <rect>
       <x>110</x>
       <y>30</y>
       <width>26</width>
       <height>22</height>
      </rect>
     </property>
     <property name="text">
      <string>...</string>
     </property>
    </widget>
   </widget>
   <widget class="QGroupBox" name="Visualizar">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>0</y>
      <width>271</width>
      <height>61</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">innerGBox-&gt;setStyleSheet(&quot;background-color: red&quot;);</string>
    </property>
    <property name="title">
     <string>Visualizar objetos</string>
    </property>
    <widget class="QCheckBox" name="flagBoxes">
     <property name="geometry">
      <rect>
       <x>10</x>
       <y>30</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Bounding Boxes</string>
     </property>
    </widget>
    <widget class="QCheckBox" name="flagLine">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>30</y>
       <width>121</width>
       <height>20</height>
      </rect>
     </property>
     <property name="text">
      <string>Lineas deteccion</string>
     </property>
    </widget>
   </widget>
   <widget class="QTableWidget" name="tableCarDetect">
    <property name="geometry">
     <rect>
      <x>710</x>
      <y>10</y>
      <width>321</width>
      <height>631</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableVLine">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>460</y>
      <width>661</width>
      <height>81</height>
     </rect>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableHline">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>570</y>
      <width>661</width>
      <height>71</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>440</y>
      <width>113</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Lineas Verticales </string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>550</y>
      <width>131</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Lineas Horizontales</string>
    </property>
   </widget>
   <widget class="QGroupBox" name="groupBox_Carcount">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>0</y>
      <width>121</width>
      <height>61</height>
     </rect>
    </property>
    <property name="title">
     <string>Contador Vehiculos</string>
    </property>
    <widget class="QLCDNumber" name="CarCount">
     <property name="geometry">
      <rect>
       <x>30</x>
       <y>30</y>
       <width>64</width>
       <height>23</height>
      </rect>
     </property>
     <property name="styleSheet">
      <string notr="true">background-color: rgb(21, 130, 209);</string>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="videoPanel">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>80</y>
      <width>671</width>
      <height>331</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QToolButton" name="start_video">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>21</y>
      <width>51</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>...</string>
    </property>
   </widget>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
